v22.2/cdc-transformations.md

```diff
@@ -0,0 +1,281 @@
+
+You can use CDC transformations to do the following:
+
+- Filter out specific rows and columns from changefeed messages to decrease the load on your downstream sink and support outbox workloads.
+- Modify data before it emits to reduce the time and operational burden of filtering or transforming data downstream.
+- Stabilize or customize the schema of your changefeed messages for increased compatibility with external systems.
+
+You can use any CockroachDB-supported SQL expression syntax that is not listed in [limitations](#limitations) to build a changefeed query.
+
+See the [Examples](#examples) section for further use cases.
+
+## Syntax
+
+There are two possible components to CDC transformations:
+
+- _Projections_ will select the columns that you want to emit data from. 
+- _Predicates_ will restrict the selected column change data based on the filters you apply.
+
+~~~ sql
+CREATE CHANGEFEED [INTO sink] [WITH options] AS SELECT projection FROM table [WHERE predicate];
+~~~
+
+Parameter        | Description
+-----------------+---------------
+`sink`             | Specify the sink URL to emit change data to. See [Changefeed Sinks](changefeed-sinks.html) for a list of supported sinks. It is also possible to run a changefeed without a sink `CREATE CHANGEFEED WITH...`, which will send changes to the active SQL session.
+`options`          | Set options on the changefeed. See the [Options](create-changefeed.html#options) table for a full list.
+`projection`       | Select the columns from which to emit data.
+`table`            | Define the table to which the columns belong.
+`predicate`        | Apply optional filters with a `WHERE` clause.
+
+For a SQL diagram of the changefeed expression syntax, see the [`CREATE CHANGEFEED`](create-changefeed.html#synopsis) page.
+
+## Limitations
+
+- It is necessary to pass the [`schema_change_policy='stop'`](create-changefeed.html#schema-policy) option in the changefeed creation statement when using the expression format. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/85143).
+- You can only apply CDC transformations on a single table in each statement.
+- Some stable functions, notably functions that return MVCC timestamps, are overridden to return the MVCC timestamp of the event.
+- You cannot [alter](alter-changefeed.html) a changefeed that uses an expression.
+
+The following are not permitted in a changefeed expression:
+
+- [Volatile functions](functions-and-operators.html#function-volatility)
+- Sub-select queries
+- [Aggregate](functions-and-operators.html#aggregate-functions) and [window functions](window-functions.html) (i.e., functions operating over many rows).
+
+## Changefeed functions
+
+{{site.data.alerts.callout_info}}
+The following functions are in preview and **subject to change** with continued development.
```

Comment:
I think maybe we should put this caveat on the syntax area as well? I know we've talked about modifying the syntax pre moving out of preview

Reply:
The whole page has the preview feature label at the top... having three or more notes on things being subject to change might be a bit too much? If we don't feel that the one at the top is enough, perhaps I could combine and make a second note box (or warning?) further down near both the syntax and functions that warns the functions and syntax are in development and subject to change. 

In theory, the feature is marked as Preview, which covers any changes. But, I understand someone could miss that and be frustrated especially with changing syntax.

Reply:
Hmm yeah that's fair, my guess is we'll still allow the old syntax anyways and just change the docs so maybe not needed